src/ol/interaction/pinchzoom.js

```diff
@@ -29,6 +29,13 @@ ol.interaction.PinchZoom = function(opt_options) {
 
   /**
    * @private
+   * @type {boolean}
+   */
+  this.constrainResolution_ = options.constrainResolution ?
+options.constrainResolution : false;
```

Comment:
I'll see about updating our linter config to catch this.  There should be two indents in a multi-line conditional expression.  E.g.
```js
var foo = bar ?
    bam : baz;
```